.devfile.yaml

```diff
@@ -43,7 +43,7 @@ components:
 # Alternatively could be removed once https://github.com/redhat-developer/odo/issues/7162 is resolved
   - name: index-generator
     container:
-      image: registry.access.redhat.com/ubi8/go-toolset:1.18
+      image: registry.access.redhat.com/ubi8/go-toolset:1.20.10-3
```

Comment:
@michael-valdron I think better to exclude this devfile from the check right?

Reply:
/hold

Reply:
> @michael-valdron I think better to exclude this devfile from the check right?

@thepetk Yes this should be excluded as there is nothing under the devfile for the build and deployment of devfile registry that needs automated patching.